recipe/meta.yaml

```diff
@@ -25,16 +25,18 @@ build:
 requirements:
   build:
     - r-base
-    - r-rcpp
+    - r-rcpp >=0.12.3
+    - r-rlang
     - r-lazyeval >=0.1.10
     - posix                # [win]
     - {{native}}toolchain  # [win]
     - gcc                  # [not win]
 
   run:
     - r-base
-    - r-rcpp
-    - r-lazyeval >=0.1.10
+    - r-rcpp >=0.12.3
+    - r-rlang
+    - libgcc  # [not win]
```

Comment:
Does it also need `- {{native}}gcc-libs # [win]` specified as a run-time dependency?